examples/package/rbxasset.lock

```diff
@@ -1,6 +1,6 @@
 [assets.package]
-assetId = "85269576874155"
+assetId = "115689279560293"
 
 [images."icon.png"]
-assetId = "108457732972751"
+assetId = "107438925431083"
```

Comment:
out of curiosity where do we get these? i see that we have a test validating it's reading in from the toml. Is it autogenerated or do we find and add it?